public/css/style.css

```diff
@@ -468,8 +473,12 @@ tr:last-child td {
   height: 56px;
   line-height: 56px;
   padding-left: 20px;
-  display: none;
   pointer-events: none;
+  transform: translateY(56px);
+  opacity: 0;
+  transition-property: transform,opacity;
```

Comment:
Hm - good question, you could use `all` instead, not sure if that has any negative implications. Shorthand with multiple transitions works like this:

```
height 0.3s ease-out, opacity 0.3s ease 0.5s;
```

but this means you have to duplicate timing etc which is less good.